src/medication-dispense/medication-dispense.resource.tsx

```diff
@@ -38,7 +38,7 @@ export function initiateMedicationDispenseBody(
   medicationRequests.map((medicationRequest) => {
     let dispense = {
       resourceType: "MedicationDispense",
-      status: "on-hold", // might need to change this to appropriate status
+      status: "completed", // might need to change this to appropriate status
```

Comment:
We still may need to tweak this, but "completed" is more accurate than "on-hold", so just updating this while it's on my mind.

Reply:
I agree with this.